geodiff/tests/testdata/2_deletes/base-deleted_A.json

```diff
@@ -1,26 +1,26 @@
 {
```

Comment:
why do we need to change json (reference) files in the tests? it will be better to compare JSON-s in test regardless of the order of elements? or is it possible to produces same output JSON files as before?